client/src/pages/ScheduleCalendar/style.js

```diff
@@ -0,0 +1,39 @@
+const ScheduleCalendarStyle = () => ({
+    scheduleCalendarContainer : {
+        display: "flex",
+        margin  : "auto",
+        width : "1000px",
+        height : "480px",
+        "-webkit-box-shadow" : "1px 1px 10px 3px rgba(196,196,196,1)",
```

Comment:
i believe you can just use `boxShadow` instead of `-webkit-box-shadow` for all modern browsers